compiler-utils/build.gradle

```diff
@@ -21,6 +21,8 @@ dependencies {
   testFixturesImplementation project(':compiler')
   testFixturesImplementation deps.dagger2.compiler
   testFixturesImplementation deps.dagger2.dagger
+  testFixturesImplementation deps.auto.value.annotations
+  testFixturesImplementation deps.auto.value.processor
```

Comment:
Necessary because this is what dagger uses when it runs to support instantiating annotations at runtime

Reply:
Good idea - 888cc04